oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -57,14 +57,21 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
       Pem.Block key = certificates.get(certKey);
       String host = request.header("Host");
 
+      Map<String, Object> claims = new LinkedHashMap<>();
+      claims.put("iss", host);
+      claims.put("aud", client.id);
+      claims.put("sub", identity.id());
+      claims.put("iat", instant.asDate());
+      claims.put("exp", new Date(accessToken.expirationTimestamp()));
```

Comment:
This JSON:

```javascript
{"iss":"example.host","aud":"123","sub":"123","iat":1494508639432,"exp":1494508639432,"name":"Pojo","email":"example@email.com","given_name":"Foo","family_name":"Bar","customerId":"::customer Id::","customerName":"::customer name::","isAdmin":false}
```
is fine as values are serialized as longs (I suppose jackson is doing it by default)